example/ios/patches/SessionDelegate.patch

```diff
@@ -0,0 +1,18 @@
+--- SessionDelegate.swifts	2021-08-10 16:26:55.000000000 -0400
++++ SessionDelegate.swift	2021-08-10 16:27:33.000000000 -0400
```

Comment:
hmm, not sure if this can be done.. we don't have access to the eventMonitor as is internal.
![image](https://user-images.githubusercontent.com/6757047/128940407-aed341c4-1873-49b6-adcf-47c02606cbcd.png)